challenges/systems/challenge-2.md

```diff
@@ -0,0 +1,294 @@
+# Summary
+
+Implement a prototype load balancer.
+
+# Rationale
+
+This exercise has two goals:
+
+* It helps us to understand what to expect from you as a developer, how you
+  write production code, how you reason about API design and how you
```

Comment:
Should we move "production"?

In the summary we say "implement a prototype" and then we say we're using that to evaluate how you write production code.

Writing code for a prototype can be vastly different than for a production system. Let's be clear about what we're looking for here.